apps/dokploy/templates/listmonk/index.ts

```diff
@@ -21,7 +21,8 @@ export function generate(schema: Schema): Template {
 
 	const mounts: Template["mounts"] = [
 		{
-			mountPath: "./config.toml",
+			filePath: "config.toml",
+			mountPath: "../files/config.toml",
```

Comment:
the mount path should be 
```suggestion
			mountPath: "config.toml",
```